rsmtool/preprocess.py

```diff
@@ -15,6 +15,22 @@ def trim(values, trim_min, trim_max):
     """
     Trim the values contained in the given vector
     `values` to `trim_min` and `trim_max`
```

Comment:
We should probably edit this for accuracy to refer to trim_max+0.49998 and trim_min-0.49998